src/screens/OnBoarding/index.tsx

```diff
@@ -0,0 +1,82 @@
+import React from 'react';
+import {StatusBar} from 'react-native';
+import Button from '~/components/Button';
+import Divider from '~/components/Divider';
+import RowBetween from '~/components/RowBetween';
+import Separator from '~/components/Separator';
+import Text from '~/components/Text';
+import useBarStyle from '~/hooks/useBarStyle';
+import {categories} from './mocks';
+
+import {Onboarding} from './styles';
+import useOnboardingController from './useOnboardingController';
+
+const OnBoardingScreen: React.FC = () => {
+  const {
+    Container,
+    Content,
+    SkipButton,
+    TopicDescription,
+    CategoriesContainer,
+    CategoriesTouchables,
+    CategoryLabel,
+  } = Onboarding;
+
+  const {handleBarStyle} = useBarStyle();
+
+  const {
+    handleCategoriesPick,
+    spacing,
+    selectedCategories,
+    handleNavigateToHomeScreen,
+  } = useOnboardingController();
+
+  return (
+    <Container>
+      <Content>
+        <StatusBar
+          backgroundColor="transparent"
+          translucent
+          barStyle={handleBarStyle}
+        />
+        <RowBetween>
+          <Text color="text">
+            App<Text color="primary">News</Text>
+          </Text>
+          <SkipButton onPress={handleNavigateToHomeScreen}>
+            <Text typography="overline">Skip</Text>
```

Comment:
The "skip" text button of the layout is with grey color. 
```suggestion
            <Text color={colors.grey} typography="overline">Skip</Text>
```

![image](https://user-images.githubusercontent.com/40049825/180662541-15efa4d4-e0a2-4b06-b8e6-3e2a5f920413.png)